pkg/router/f5/f5.go

```diff
@@ -459,6 +459,35 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 }
 
 //
+// iControl REST resource helper methods.
+//
+
+// iControlResourceUriPath returns a fully qualified `escaped` resource URI path component that can be used in the iControl REST calls.
```

Comment:
What does "fully qualified" mean? On first reading, it looks to me as though you mean a "fully qualifier" "URI path component", implying that the result is the full path that comes after the hostname in the URL, which is not the case.  On a subsequent reading, I think you mean "fully qualified" "resource", meaning the resource is fully qualified by specifying the partition—i.e., it's an absolute path—but then it looks like "escaped" modifies "resource", not "URI".

If you mean that `iControlResourceUriPath` ensures that the escaped path is absolute, can we just say as much? Something along the lines of "returns an encoded absolute path (including the partition) for the resource, which can be used as a URI path component in iControl REST calls."

Is there a reason to quote "escaped" that way?

My comments for `iControlResourceUriPath` apply to `iControlVserverUriPath` and `escapeiControlUriPath` as well.